src/Cell.tsx

```diff
@@ -56,7 +53,6 @@ function Cell<R>({
   function handleCellDoubleClick(e: React.MouseEvent<HTMLDivElement>) {
     e.stopPropagation();
```

Comment:
Why do we do this? It'll prevent the double click event to be triggered on the Row `div`.

Reply:
I am not sure
https://github.com/adazzle/react-data-grid/commit/1c273ed8b2a8b8eae01496b4da51deec496cc821#diff-d22f3ef92a32a6dcd0ca1ea0fe869cebR83